ros2node/ros2node/api/__init__.py

```diff
@@ -24,6 +24,14 @@
 TopicInfo = namedtuple('Topic', ('name', 'types'))
 
 
+def _is_hidden_name(name):
+    # note, we're assuming the hidden node prefix is the same for other hidden names
+    return (
+        name and
```

Comment:
What is this check needed for?

Reply:
It was copied from the [hidden node logic](https://github.com/ros2/ros2cli/blob/c17f7dbb7fb8d15046b8719e22a2fb71e6c3e9f8/ros2node/ros2node/api/__init__.py#L54), but I guess we can't have empty topic names so this shouldn't be needed.

Reply:
Removed 2367227